networker/networker.go

```diff
@@ -1134,18 +1151,17 @@ func (netw *Combined) setMesh(
 		}
 	}
 
-	if !netw.isVpnSet {
-		if err = netw.policyRouter.SetupRoutingRules(
-			netw.mesh.Tun().Interface(),
-			false,
-		); err != nil {
-			return fmt.Errorf(
-				"setting routing rules: %w",
-				err,
-			)
-		}
-		routingRulesSet = true
+	if err = netw.policyRouter.SetupRoutingRules(
+		netw.mesh.Tun().Interface(),
+		false,
+		netw.enableLocalTraffic,
```

Comment:
I think my comment about this always being `true` is still valid. It's subjective, so change only if you agree, but I think this parameter being `netw.enableLocalTraffic` instead of just `true` slightly gives wrong sense that this is dynamic when it isn't :D

Reply:
When mesh connect peer (peer allows lan access, enableLocalTraffic=false) -> with your laptop go outside building i.e. switch to mobile network -> Combined.Reconnect(...)->Combined.refreshVPN(...)->Networker.setMesh(...)->policyRouter.SetupRoutingRules(...., netw.enableLocalTraffic)

If you set it to `true` always then you will loose what was set during mesh peer connect, as it does not replay `mesh peer connect` during network change event.
The same should be done for `ipv6enabled` parameters as well, I guess.